src/sql/plot.py

```diff
@@ -46,6 +48,7 @@ def _summary_stats(conn, table, column, with_=None):
     )
 
     query = template.render(table=table, column=column)
+    query = conn._transpile_query(query)
```

Comment:
I don't think we need to transpile. we write the queries so they work with duckdb out of the box. so in this case, they should work without transpiling. @tonykploomber, am I right? or are we missing the transpile step here?

Reply:
Ok, it seems like queries are getting transpiled every time we call `execute` [here](https://github.com/ploomber/jupysql/blob/e5276711ea408878a62feca46c017a37243da9c3/src/sql/connection.py#L654) with the help of [`_prepare_query`](https://github.com/ploomber/jupysql/blob/e5276711ea408878a62feca46c017a37243da9c3/src/sql/connection.py#L625)

So, I will remove the line as it is redundant, but queries are getting transpiled, every time we call `execute`

Reply:
that looks like a problem. what I don't understand about the current implementation is why duckdb + sqlalchemy works but duckdb + dbapi doesn't. based on my earlier experiment, I noticed that duckdb + dbapi produced invalid SQL (possibly due to transpilation). so if queries are always transpiled, why does duckdb + sqlalchemy doesn't break?

Reply:
ok, I understand now